### DISCUSSION TITLE
dependabot

### DISCUSSION THREAD

**Discussion by pierresisson** [NostrGit/NostrGit#74] [General]
what do you think about https://github.com/dependabot ?

**Comment by pierresisson** [comment]
@PeerRich i can't add it, can you ? 
in settings
![image](https://user-images.githubusercontent.com/79455860/223418648-88f3efc5-d06b-46b7-9ca9-a4e72d08c1bb.png)
![image](https://user-images.githubusercontent.com/79455860/223418707-8a7453f0-6e0f-4ce2-8314-77f13a26d334.png)

**Reply by cypherhoodlum** [reply]
Not familiar with it but seems like a fantastic tool if it's smart enough to not cause any breaking changes. Too much to ask for a bot like this? Do you recommend it @pierresisson?

**Reply by cypherhoodlum** [reply]
Let's remove it if it causes any headaches. Remaining hopeful :smile:
